packages/@webex/plugin-cc/src/types.ts

```diff
@@ -1,5 +1,6 @@
 import {CallingClientConfig} from '@webex/calling';
 import * as Agent from './services/agent/types';
+import * as Contact from './services/Task/types';
```

Comment:
Could we stick to the convention of naming the folder in all small letters?
```suggestion
import * as Contact from './services/task/types';
```

Reply:
Addressed